styles.css

```diff
@@ -93,3 +84,215 @@ header {
+}
+
+.tonic {
+  font-family: 'Poppins', sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  text-align: left;
+  color: #172b4d;
+  font-style: normal;
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.canopy {
+  font-family: 'Poppins', sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  text-align: left;
+  color: #344563;
+}
+
+.cano {
+  font-family: 'Poppins', sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  text-align: left;
+  color: #7a869a;
+  display: inline-flex;
+  gap: 30px;
+  align-items: center;
+}
+
+.list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.button1 {
+  font-family: 'Poppins', sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  width: 115px;
+  color: #396df2;
+  background-color: white;
+  border: #396df2 solid 2px;
+  border-radius: 10px;
+  height: 50px;
+}
+
+.button1 :hover {
+  color: #c16e3a;
+}
+
+.button2 {
+  font-family: 'Poppins', sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  width: 154px;
+  color: #396df2;
+  background-color: white;
+  border: #396df2 solid 2px;
+  border-radius: 10px;
+  height: 50px;
+}
+
+.button:hover {
+  color: #c16e3a;
+}
+
+.p {
+  font-family: 'Poppins', sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  color: #344563;
+  padding: 0;
+  margin: 5px;
+}
+
+.lang {
+  display: flex;
+  list-style-type: none;
+  text-decoration: none;
+  margin: 0;
+  padding: 0;
+  margin-right: 10px;
+  gap: 10px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+}
+
+.lang li {
+  background-color: #ebebff;
+  border-radius: 10px;
+  color: #6070ff;
+  padding: 4px 12px;
+  margin-bottom: 20px;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  background-image: url(images/Base\ about\ my\ self.png);
+  margin: 0;
+  padding: 0;
+}
+
+.myself {
+  color: #172b4d;
+  width: 327px;
+  height: 35px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  flex: none;
+  padding-bottom: 20px;
+}
+
+.AboutMyself {
+  font-family: 'Poppins', sans-serif;
+  font-size: 18px;
+  font-weight: 580;
+  line-height: 24px;
+  text-align: left;
+  color: #344563;
+  width: 327px;
+  height: 147px;
+}
+
+.langu {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-right: 10px;
+  margin-left: -34px;
+  padding: 0;
+}
+
+.langu ul {
+  display: flex;
+  flex-direction: column;
+  list-style-type: none;
+}
+
+.languag li img {
+  width: 100%;
+}
+
+.abo {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin-left: 10px;
+}
```

Comment:
- [ ] You did good work but still, your design has some little issues. Kindly adhere to the project requirements as you can see in the attached image down below.

![image](https://user-images.githubusercontent.com/66476329/235255189-1f90f047-2db8-44d5-a0b3-c6251b1fd8cb.png)

- [ ] Kindly take a look again at your design and Figma design. Below the table left-hand-side is the Figma design and the right-hand side is your design. 

<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-04-28 17-15-44](https://user-images.githubusercontent.com/66476329/235145079-2556c9b2-05e0-4641-a58f-c707d995a90b.png)

</td>
<td>

![Screenshot from 2023-04-29 02-25-21](https://user-images.githubusercontent.com/66476329/235256732-6d7e24bd-0e3b-4dc2-89fe-1458993c6c3c.png)

</td>
</tr>
</table>